### PR TITLE
Various enhancements

### PR DESCRIPTION
**Pull request by fh-ms**

### CODE REVIEW

File: persistence/binary/src/main/java/org/eclipse/serializer/persistence/binary/org/eclipse/serializer/collections/BinaryHandlerEqHashTable.java [hunk 1]
```diff
@@ -15,15 +25,6 @@
  */
```
Comment (zdenek-jonas) [inline persistence/binary/src/main/java/org/eclipse/serializer/persistence/binary/org/eclipse/serializer/collections/BinaryHandlerEqHashTable.java:25]: Bad order... Licence first

File: persistence/binary/src/main/java/org/eclipse/serializer/persistence/binary/org/eclipse/serializer/collections/BinaryHandlerHashTable.java [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
+import org.eclipse.serializer.collections.HashTable;
+import org.eclipse.serializer.persistence.binary.types.AbstractBinaryHandlerCustomCollection;
+import org.eclipse.serializer.persistence.binary.types.Binary;
+import org.eclipse.serializer.persistence.types.Persistence;
+import org.eclipse.serializer.persistence.types.PersistenceFunction;
+import org.eclipse.serializer.persistence.types.PersistenceLoadHandler;
+import org.eclipse.serializer.persistence.types.PersistenceReferenceLoader;
+import org.eclipse.serializer.persistence.types.PersistenceStoreHandler;
+
 /*-
```
Comment (zdenek-jonas) [inline persistence/binary/src/main/java/org/eclipse/serializer/persistence/binary/org/eclipse/serializer/collections/BinaryHandlerHashTable.java:12]: Bad order licence first, second imports

File: persistence/binary/src/main/java/org/eclipse/serializer/persistence/binary/org/eclipse/serializer/collections/BinaryHandlerEqHashTable.java (hunk 1 above)
Comment (fh-ms) [reply persistence/binary/src/main/java/org/eclipse/serializer/persistence/binary/org/eclipse/serializer/collections/BinaryHandlerEqHashTable.java:15]: Fixed

File: persistence/binary/src/main/java/org/eclipse/serializer/persistence/binary/org/eclipse/serializer/collections/BinaryHandlerHashTable.java (hunk 2 above)
Comment (fh-ms) [reply persistence/binary/src/main/java/org/eclipse/serializer/persistence/binary/org/eclipse/serializer/collections/BinaryHandlerHashTable.java]: Fixed

**Review by zdenek-jonas — APPROVED** [review]

**Pull request merged**